src/util.py

```diff
@@ -31,6 +31,11 @@ def github_url(prefix, owner=None, repo=None, detail=None, params=None):
 def content_url(owner, repo, version, path):
   return 'https://raw.githubusercontent.com/%s/%s/%s/%s' % (owner, repo, version, path)
 
+def npm_registry_url(scope, package):
```

Comment:
It's a mixed bag at the moment. I'm avoiding any renames to try minimize just how many lines have to change at this stage and it helps to keep track of what's actually been looked carefully at already. Maybe at the end we can clean it up, but its worth nothing some are actually owner/repo still.